src/ooxml-package/ooxml-package.ts

```diff
@@ -178,6 +180,14 @@ export class OOXMLPackage {
     }
   }
 
+  /**
+   * Removes the OOXML package from the tree view.
+   */
+
+  async removePackage(): Promise<void> {
+    await this.removeOOXMLPackage(this.ooxmlFilePath);
```

Comment:
It might be cleaner to do this instead of passing in the removeOOXMLPackage function

`await ExtensionUtilities.dispatch(new RemoveOOXMLCommand(this.treeView.getRootFileNode()));`

Reply:
done